skimage/measure/fit.py

```diff
@@ -776,6 +779,16 @@ def ransac(data, model_class, min_samples, residual_threshold,
             True,  True,  True,  True,  True,  True,  True,  True,  True,
             True,  True,  True,  True,  True], dtype=bool)
 
+    Les us present an example of transforming this function requesting mimimal number
```

Comment:
I would suggest changing this example to use `ransac` directly. We don't need to teach users to write wrapper functions, but rather, we need to propose alternative ways to use our functions, and it is up to them whether they want to wrap. Here's my attempt:

```
The argument `min_samples` is an absolute number. If instead we want to use a proportion
of the full dataset, we can multiply by the length of the data and cast to int:

>>> model_class = SimilarityTransform
>>> min_sample_ratio = 0.5
>>> min_samples = int(min_sample_ratio * len(src))
>>> model, inliers = ransac((src, dst), model_class, min_samples, ...)
```

Reply:
nice, I have changed the second example accordingly